gadgets/weather/src/wttr.rs

```diff
@@ -0,0 +1,90 @@
+use std::{fmt::Display, str::FromStr};
```

Comment:
This code looks familiar... :thonk:

:P

Reply:
oh yeah, I took it from tarchboard, want me to link it somehow? (Or add you as a coauthor to the commit?)

Reply:
nah dwai